wallaby.conf.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(wallaby) {
     files: [
       'packages/**/src/**/*.+(js|jsx|json|snap|css|less|sass|scss|jpg|jpeg|gif|png|svg)',
       '!packages/**/src/**/*.test.js?(x)',
-      'vendor/**',
+      '!packages/utils/**', // This is a test
       '!packages/**/node_modules/**',
```

Comment:
`// This is a test` seems odd in this context. Is this something you were testing and it was accidentally left in? Or are you intentionally excluding a package containing tests?

Reply:
No, this pacakge is a test and should not be included in source for wallaby.

The wallaby configuration specifies that files: [] is for source and tests: [] is for tests. It is not being included in files because it is a test.

Reply:
Can you update the comment to make it obvious

Reply:
:ok_hand: